en/docs/SetupAndInstall/UpgradingWSO2APIManager/upgrading-from-250-to-300.md

```diff
@@ -3,350 +3,1678 @@
 The following information describes how to upgrade your API Manager server **from APIM 2.5.0 to 3.0.0** .
 
 !!! note
-To upgrade **from a version older than 1.8.0** , follow the instructions in the document that was released immediately after your current release and upgrade incrementally.
+    To upgrade **from a version older than 1.8.0** , follow the instructions in the document that was released immediately after your current release and upgrade incrementally.
```

Comment:
We have moved this part to Upgrading process document. Shall we follow the same format in the 2.6 to 3.0 doc?

Reply:
Fixed